skimage/restoration/tests/test_denoise.py

```diff
@@ -153,29 +153,29 @@ def test_denoise_tv_bregman_3d():
 
 
 def test_denoise_bilateral_2d():
-    img = checkerboard_gray.copy()
+    img = checkerboard_gray.copy()[:50,:50]
     # add some random noise
     img += 0.5 * img.std() * np.random.rand(*img.shape)
     img = np.clip(img, 0, 1)
 
     out1 = restoration.denoise_bilateral(img, sigma_color=0.1,
-                                         sigma_spatial=20, multichannel=False)
```

Comment:
@soupault If using sigma_spatial=5 and 10, I get  this:
```
FAIL: skimage.restoration.tests.test_denoise.test_denoise_bilateral_color
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/opt/anaconda/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/alex/github/scikit-image/skimage/restoration/tests/test_denoise.py", line 182, in test_denoise_bilateral_color
```

That only happens with those values; it doesn't happen in the version in this PR, of course.
    assert out1[30:45, 5:15].std() > out2[30:45, 5:15].std()
AssertionError